kratos/geometries/nurbs_curve_geometry.h

```diff
@@ -330,6 +329,27 @@ class NurbsCurveGeometry : public Geometry<TPointType>
         return rResult;
     }
 
+    /**
+     * @brief Returns the local coordinates of a given arbitrary point
+     * @param rResult The vector containing the local coordinates of the point
+     * @param rPoint The point in global coordinates
+     * @return The vector containing the local coordinates of the point
+     */
+    CoordinatesArrayType& PointLocalCoordinates(
```

Comment:
this function does NOT do projection
The input should already be on the surface

Reply:
It projects from 3D space to 2D space. So the actual behavior is exactly this.

Reply:
from physical to parameter space?

Reply:
Exactly

Reply:
ok then it does what it is supposed to do :)

then the name is misleading
maybe use sth like `ProjectionNurbsGeometryUtilities::MapFromGlobalToParameterSpace`

Reply:
This is indeed the Newton-Raphson. Within the PointLocalCoordinates() function is some more logic to use the Newton-Raphson properly. If we realize later that the algorithm is not stable enough for our problems we would need to modifiy the PointLocalCoordinates function, however the Newton-Raphson can be untouched. So I would prefer to keep the name, if that's ok for you?